BackofficeBundle/DependencyInjection/OpenOrchestraBackofficeExtension.php

```diff
@@ -36,10 +23,9 @@ public function load(array $configs, ContainerBuilder $container)
         $config = $this->processConfiguration($configuration, $configs);
 
         $this->updateBlockParameter($container, $config);
-        if (empty($config['front_languages'])) {
-            $config['front_languages'] = array('en' => 'open_orchestra_backoffice.language.en', 'fr' => 'open_orchestra_backoffice.language.fr', 'de' => 'open_orchestra_backoffice.language.de');
-        }
+
         $fixedAttributes = array_merge($config['fixed_attributes'], array('component', 'submit', 'label', 'class', 'id', 'maxAge'));
```

Comment:
I'd put all that 'static' settings in a config files instead of directly in the php